RiotSwiftUI/Modules/UserSessions/Common/UserSessionInfo.swift

```diff
@@ -79,6 +79,8 @@ struct UserSessionInfo: Identifiable {
         case unverified
         /// The session has been verified.
         case verified
+        /// A session which cannot be never verified due to lack of crypto support
+        case permanentlyUnverified
     }
```

Comment:
what about adding a var in the VerificationState called 
```
isUnverified: Bool {
 return self == .verified || self == .permanentlyUnverified 
 }
```

Reply:
It makes sense!
Applied here -> https://github.com/vector-im/element-ios/pull/7083/commits/6b01a7236cbded0758fe4f7fa9e781a57c0f0f68